### ISSUE TITLE
MaterialSplitPanel handlers for dragstop events

### ISSUE THREAD

**Issue by gkresic** [GwtMaterialDesign/gwt-material-addins#231]
touchsplitter.js throws standard `dragstart` *non-standard* `dragstop` event every time its splitters are dragged around. Since `dragstart` is standard event, there are already support for event handlers via `MaterialWidget.addDragStartHandler`. However, `dragstop` is unsupported.

Currently, I'm subscribing to these events using JQuery:

```
mySplitPanel.$this().bind("dragstop", this::onPanelResize);
```

I could provide a PR for this support, but first I would like to ask how these non-standard events should be handled in GMD? Should I introduce a new `GwtEvent`-derived event? Or should I catch `dragstop` and throw `DragEndEvent` instead? Why GMD provides its own drag events like `DragStartEvent` instead of using `com.google.gwt.event.dom.client.DragStartEvent`?

**Comment by kevzlou7979** [comment]
@gkresic Yes sure, we will accept PR and review. About the GMD Events - yes we have added a prebuilt events ```DragStartEvent``` which you can use, just create an interface ```HasSplitPanelHandler``` then create each method to be able to implement on MaterialSplitPanel, try to study this https://github.com/GwtMaterialDesign/gwt-material/blob/master/gwt-material/src/main/java/gwt/material/design/client/base/HasSideNavHandlers.java

**Comment by kevzlou7979** [comment]
https://github.com/colelawrence/Touch-Splitter-jQuery/issues/14
